bot/utils/persistent_scheduling.py

```diff
@@ -68,8 +68,10 @@ async def to_dict(self) -> t.Dict[CacheKey, Arrow]:
 
     async def reschedule_task(self, task_id: CacheKey) -> None:
         """Reschedule the task according to `task_factory` and `task_id`."""
-        time = (await self.get(task_id)).datetime
-        self._scheduler.schedule_at(time, task_id, coroutine=self._to_schedule(task_id))
+        time = await self.get(task_id)
+        if not time:
+            return
```

Comment:
Do you not think it's worth raising an error?

Reply:
edec063